ISISComputingGroup/EPICS-ioc
# Fix IOC system tests

### Description of work

Fixed IOC system tests by making the lakeshore more responsive when setting `PMRS`

### To test

* Run system tests for lakeshore 460, confirm they fail
* Rebuild lakeshore 460
* Confirm tests now pass

---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
